website/content/commands/docs.mdx

```diff
@@ -41,8 +41,8 @@ The flags can change which plugins are listed and in which format.
 
 #### Command Options
 
-- `-builtin` - Show documentation on all builtin plugins
-- `-markdown` - Show documentation in markdown format
+- `-builtin` - Show documentation on all builtin plugins The default is false.
+- `-markdown` - Show documentation in markdown format The default is false.
```

Comment:
I see a handful of these cases where the docs don't end in a period, and this new auto-generated part doesn't start with one. I'm fine with just fixing the isolated cases and adding a period at the end instead of more code that checks for it, but we could do that as well

Reply:
True....I'll do a quick scan and fix these. It might be nice to have the auto-check for future docs in the case where we forget to add periods.

Reply:
Done in [4bcbc1c](https://github.com/hashicorp/waypoint/pull/3501/commits/4bcbc1ce28c7869cfcd015def8659ebdb299e175)!